platform/src/server/http/ssl_config.ts

```diff
@@ -64,10 +67,35 @@ export class SslConfig {
     this.enabled = config.enabled;
     this.key = config.key;
     this.certificate = config.certificate;
-    this.certificateAuthorities = config.certificateAuthorities;
+    this.certificateAuthorities = this.initCertificateAuthorities(config.certificateAuthorities);
     this.keyPassphrase = config.keyPassphrase;
-    this.keystore = config.keystore;
     this.cipherSuites = config.cipherSuites;
     this.supportedProtocols = config.supportedProtocols;
   }
+
+  /**
+   * Options that affect the OpenSSL protocol behavior via numeric bitmask of the SSL_OP_* options from OpenSSL Options.
+   */
+  getSecureOptions() {
```

Comment:
This seems pretty different from the express stuff, and it's not clear what functionality from express it is replacing. Can you explain a bit what this is doing? Or maybe just point out in this diff where the corresponding express stuff is being removed so I can research on my own?

Reply:
> This seems pretty different from the express stuff

What express stuff do you mean? We [never supported `secureOptions` in new platform](https://github.com/elastic/kibana/blob/new-platform/platform/src/server/http/http_server.ts#L77), so it's almost a copy-paste of [what we have in master](https://github.com/elastic/kibana/blob/97df91f36f991cc38cdd0d911618a5bba7da52ac/src/server/http/secure_options.js).

Reply:
That's perfect. I was mostly just trying to figure out why you added this code in the first place.